Datasets/data-connect-dataset-user1.md

```diff
@@ -65,7 +65,7 @@ The User_v1 dataset can be joined with other datasets that can be mapped to the
 | givenName  | string  | The given name (first name) of the user.  | No  | None  |
 | hireDate  | string  | The hire date of the user. The Timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 would look like this: '2014-01-01T00:00:00Z'.  | No  | None  |
 | id  | string  | The unique identifier for the user. Inherited from directoryObject.  | No  | None  |
-| imAddresses  | string  | The instant message voice over IP (VOIP) session initiation protocol (SIP) addresses for the user. *Format:*`ARRAY<STRING>.`  | No  | None  |
+| imAddresses  | array  | The instant message voice over IP (VOIP) session initiation protocol (SIP) addresses for the user. *Format:*`ARRAY<STRING>.`  | No  | None  |
 | interests  | string  | A list of interests for the user. *Format:*`ARRAY<STRING>.`  | No  | None  |
```

Comment:
"interests" should be an array instead of string

Reply:
fixed